dataprofiler/profilers/profile_builder.py

```diff
@@ -158,12 +159,14 @@ def __add__(self, other):
 
         merged_profile.name = self.name
         merged_profile._update_base_stats(
-            {"sample": self.sample, 'sample_size': self.sample_size,
+            {"sample": self.sample, "sample_ids": self.sample_ids,
+             "sample_size": self.sample_size,
              "null_count": self.null_count,
              "null_types": copy.deepcopy(self.null_types_index)}
         )
         merged_profile._update_base_stats(
-            {"sample": other.sample, 'sample_size': other.sample_size,
+            {"sample": other.sample, "sample_ids": other.sample_ids,
+             "sample_size": other.sample_size,
              "null_count": other.null_count,
              "null_types": copy.deepcopy(other.null_types_index)}
```

Comment:
@gme5078 and @ChrisWallace2020 this doesn't seem correct, at least without some modifications...

For instance: 
```
df_series1 = [None, 1, 2, 3]
df_series2 = [0, None, 2, 3]
profile1 = StructuredDataProfile(df_series1)
profile2 = StructuredDataProfile(df_series2)
profile3 = profile1 + profile2
profile3.profile['null_types'] # Will be [0, 1]
```

IMO should be [0, 5] because the indices should be an extension of the dataset. Currently, the null locations will be [0, 1]

Reply:
See issue #167